installing/installing_gcp/installing-gcp-user-infra-vpc.adoc

```diff
@@ -71,6 +71,11 @@ include::modules/installation-deployment-manager-vpc.adoc[leveloffset=+3]
 include::modules/installation-user-infra-generate.adoc[leveloffset=+1]
 
 include::modules/installation-initializing-manual.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
```

Comment:
This and two others added this link, but didn't remove the module that used it (because it didn't exist in this assembly). 

Since the previous module doesn't reference the config paramters section, should this link not actually be added after all? (Same for other two assemblies that do the same)

Reply:
> Since the previous module doesn't reference the config paramters section, should this link not actually be added after all? (Same for other two assemblies that do the same)

- In the case of this assembly, `installation-initializing-manual.adoc` (Manually creating the installation configuration file) instructs users to modify the `install-config.yaml` file. (step 2).
- I believe the other two files you are referring to are `installing-gcp-user-infra.adoc` and `installing-restricted-networks-gcp.adoc`. Both of which include `installation-initializing.adoc` (Creating the installation configuration file), which instruct users to modify the `install-config.yaml` file. (step 2)

In all cases, IMO, the fact that the installation configuration parameters were not included in the assembly was an oversight, which is why I added the xref.